### DISCUSSION TITLE
Beatmap title/artist click doesn't search for the original language title/artist

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#29823] [Q&A] [answered]
After last update on beatmap title/artist click behaviour it doesn't search in original language anymore and always use the romanised version. This doesn't match the web behaviour, as web always search for the non-romanised version as it should (when you have the "show beatmap metadata in original language" option enabled, obviously), and I wanted that lazer could replicate this behaviour too.

Also, this isn't a purely cosmetic change, as some beatmaps has a different romanised version for the same artist.
Examples:
- https://osu.ppy.sh/beatmapsets/2001782
- https://osu.ppy.sh/beatmapsets/818349

**Comment by Joehuu** [comment, marked as answer]
https://github.com/ppy/osu/pull/29913
